.env.example

```diff
@@ -11,7 +11,7 @@ DB_PASSWORD=secret
 
 CACHE_DRIVER=file
 SESSION_DRIVER=file
-QUEUE_DRIVER=sync
+QUEUE_DRIVER=database
```

Comment:
You both have good points let's just remember we are making Cachet easy to install so if we make it sync by default we need to provide tools to change it. Like a setting that when you change it creates the migrations for you and automatically migrate, or composer install redis. What do you think?